articles/libraries/index.md

```diff
@@ -83,18 +64,14 @@ The most commonly used SDKs have reference documentation.
   </li>
 </ul>
 
-### SDK Support Table
-
-Below are listed all of the SDKs that are available for Auth0, with GitHub links and their support status.
+### SDK links and support status
 
 <%= include('../_includes/_libraries_support_sdks') %>
 
-
-### Framework/Platform Integration SDK Support Table
+### Framework/Platform integration SDK support status
```

Comment:
This is confusing to read (not that you wrote it, but we might as well fix now).

I wonder if we could say `### Platform integration repositories and support status` or something. Or we can just throw them all in under SDKs, but they were separated for a reason.

Reply:
Ok - I like that change.